config/peerpods/podvm/bootc/podvm-git-buildconfig.yaml

```diff
@@ -0,0 +1,32 @@
+# This will build according to git source and push to the local cluster's registry:
+# image-registry.openshift-image-registry.svc:5000/openshift-sandboxed-containers-operator/podvm-bootc
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: podvm-bootc
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  lookupPolicy:
+    local: false
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: bootc-image-build
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: podvm-bootc:latest
+  source:
+    contextDir: "config/peerpods/podvm/bootc"
+    git:
+      uri: 'https://github.com/openshift/sandboxed-containers-operator.git'
+      ref: devel
```

Comment:
When reading the current code, I noticed a few references like this, which we might forget to update during the releases. Maybe is time to use a version.yaml file, parse with jq and concentrate the references there?